pom.xml

```diff
@@ -471,13 +471,6 @@
         <groupId>com.clearspring.analytics</groupId>
         <artifactId>stream</artifactId>
         <version>2.7.0</version>
-        <exclusions>
-          <!-- Only HyperLogLogPlus is used, which doesn't depend on fastutil -->
-          <exclusion>
-            <groupId>it.unimi.dsi</groupId>
-            <artifactId>fastutil</artifactId>
```

Comment:
We need to not bring fastutil back into the build. It is massive, and caused packaging problems. Let's find another answer.